src/ext/java/org/opentripplanner/ext/siri/TimetableHelper.java

```diff
@@ -895,4 +873,35 @@ public static Optional<PickDrop> mapPickUpType(
       case PASS_THRU -> Optional.of(CANCELLED);
     };
   }
+
+  private static ZonedDateTime getServiceDate(
```

Comment:
I think you could return a LocalDate here, as now the return value is always converted to one when used.

Reply:
Agree